sky/spot/controller.py

```diff
@@ -36,6 +38,14 @@ class SpotController:
 
     def __init__(self, job_id: int, task_yaml: str,
                  retry_until_up: bool) -> None:
+
+        # Re-initialize the logger, as this is a new process.
+        # Using the same logger will cause the log not being printed to the
+        # console.
+        # Create a logger for this process
+        self.logger = sky_logging.init_logger(
+            f'{logger.name}.controller_process')
```

Comment:
```suggestion
            f'{logger.name}.SpotController')
```
Nit: to avoid confusion with a potential nested module called `controller_process`. This naming convention suggests that SpotController is not a module.

Reply:
Ahh, after trying a bit more, it seems the problem is not related to this multiprocessing. I reverted to the original one, but only keep the `__name__` change for getting the global logger.